src/components/App.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+import calculate from '../logic/calculate';
+import './styles/App.css'
+
+class App extends React.Component {
+
+  state = { total: null, next: null, operation: null };
```

Comment:
### Related to division by zero

Keeping a local state called "error" will help us handle 'division by zero' error more properly.